### ISSUE TITLE
[beA] beA-Entwurf nicht versendbar 

### ISSUE THREAD

**Issue by librelegal** [jlawyerorg/j-lawyer-org#1574]
beA-Nachrichten können entworfen und als Entwurf im Ordner Entwürfe gespeichert werden. Entwürfe können aber nicht per j-lawyer versandt werden, da kein Senden-Button beim Öffnen eines Entwurfs angezeigt wird. Es gibt nur buttons für "Antworten", "Allen  Antworten" und "Weiterleiten".

![image](https://user-images.githubusercontent.com/37673969/156193429-d77f4220-f63e-4771-881c-3b01fc424f77.png)

Es kommt dabei nicht darauf an,WER den Entwurf erstellt hat (Rechtsanwältin oder nicht-anwaltlicher Mitarbeiter, der nicht selbst zum Versand befugt wäre) oder WIE der Entwurf erstellt wurde (per j-lawyer oder per beA-Weboberfläche, Letztere lassen sich auch in j-lawyer öffnen, aber eben nur per Weboberfläche versenden). 

Per j-lawyer erstellte Entwürfe können aber in der beA-Weboberfläche durch den Berechtigten versandt werden.

Der bug tritt unter Linux und unter Windows 10 auf. 

Aus meiner Sicht ist die Funktionalität der Erstellung von Entwürfen durch nicht-anwaltliche Mitarbeiter von großer Bedeutung für die Nutzung.

**Comment by j-dimension** [comment]
Ja, das ist korrekt. Das Speichern als Entwurf ist eher dazu gedacht, dass man innerhalb der beA-Weboberfläche nachträglich signieren und versenden kann. Insofern handelt es sich hier mehr um eine Erweiterung, keinen Bug.
